src/wallet/wallet_rpc_server.cpp

```diff
@@ -60,6 +60,7 @@ using namespace epee;
 #define MONERO_DEFAULT_LOG_CATEGORY "wallet.rpc"
 
 #define DEFAULT_AUTO_REFRESH_PERIOD 20 // seconds
+#define REFRESH_INFICATIVE_BLOCK_CHUNK_SIZE 256    // just to split refresh in separate calls to play nicer with other threads
```

Comment:
Was inficative (not a word) meant to be indicative? Even indicative seems like an odd name choice, it's more like a default.

Reply:
Yes, it was. IIRC that value is only respected by monerod if the blocks aren't too big, so it's only indicative. Maybe "max" would have been better.
I won't bother changing it here since it's a copy of the master patch, which is already merged.